cli/command_snapshot_list.go

```diff
@@ -56,6 +57,7 @@ func (c *commandSnapshotList) setup(svc appServices, parent commandParent) {
 	cmd.Flag("reverse", "Reverse sort order").BoolVar(&c.reverseSort)
 	cmd.Flag("all", "Show all snapshots (not just current username/host)").Short('a').BoolVar(&c.snapshotListShowAll)
 	cmd.Flag("max-results", "Maximum number of entries per source.").Short('n').IntVar(&c.maxResultsPerPath)
+	cmd.Flag("max-sources", "Maximum number of sources.").Short('s').IntVar(&c.maxSources)
```

Comment:
that's not the right term, "source" generally refers to a source directory, what you're doing is limiting the number of snapshot manifests.

Reply:
also, this would need to have tests before we can merge

Reply:
Changed and added a test.